openmsupply/open-msupply
# Update f/e dependencies : part 3

Fixes #715 (partly)

# 👩🏻‍💻 What does this PR do? 
 
Updates electron - and a few extra bits. Electron isn't working on develop branch, so this is a good catch up.

# 🧪 How has/should this change been tested? 

Test that electron runs ok `yarn electron:start`

Then the usual
- run tests
- run pre-commit lint
- check the site builds
- check that storybook runs & builds
- run the site and check that it's working 

## 💌 Any notes for the reviewer?


## 📃 Documentation

_No user facing changes_
